test/function/functions_test.cpp

```diff
@@ -97,22 +95,21 @@ TEST_F(FunctionsTests, FuncCallTest) {
   txn_manager.CommitTransaction(txn);
 
   TestingSQLUtil::ExecuteSQLQuery("CREATE TABLE test(a DECIMAL, s VARCHAR);");
-
   TestingSQLUtil::ExecuteSQLQuery("INSERT INTO test VALUES (4.0, 'abc');");
 
   std::vector<ResultValue> result;
   std::vector<FieldInfo> tuple_descriptor;
   std::string error_message;
   int rows_affected;
 
-  TestingSQLUtil::ExecuteSQLQuery("SELECT SQRT(a), SUBSTR(s,1,2) FROM test;",
-                                  result, tuple_descriptor, rows_affected,
-                                  error_message);
-  EXPECT_EQ(1, result[0].size());
-  EXPECT_EQ('2', result[0][0]);
-  EXPECT_EQ(2, result[1].size());
-  EXPECT_EQ(result[1][0], 'a');
-  EXPECT_EQ(result[1][1], 'b');
+//  TestingSQLUtil::ExecuteSQLQuery("SELECT SQRT(a), SUBSTR(s,1,2) FROM test;",
+//                                  result, tuple_descriptor, rows_affected,
+//                                  error_message);
+//  EXPECT_EQ(1, result[0].size());
+//  EXPECT_EQ('2', result[0][0]);
+//  EXPECT_EQ(2, result[1].size());
+//  EXPECT_EQ(result[1][0], 'a');
+//  EXPECT_EQ(result[1][1], 'b');
```

Comment:
Why is this commented out?

Reply:
Did it accidentally. Already uncommented it.